stl/debugger/STL.natvis

```diff
@@ -1651,4 +1661,88 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </Expand>
   </Type>
 
+  <Type Name="std::default_sentinel_t">
+    <DisplayString>default sentinel</DisplayString>
+  </Type>
+
+  <Type Name="std::unreachable_sentinel_t">
+    <DisplayString>unreachable</DisplayString>
+  </Type>
+
+  <Type Name="std::ranges::empty_view&lt;*&gt;">
+    <DisplayString>{{ size=0 }}</DisplayString>
+  </Type>
+
+  <Type Name="std::ranges::single_view&lt;*&gt;">
+    <DisplayString>{{ size=1 }}</DisplayString>

```

Comment:
I think it will be more usefull to see the val
```suggestion
    <DisplayString>{ _Val._Val }</DisplayString>

```

Reply:
Maybe this to emphasize that it's a range:
```suggestion
    <DisplayString>{{ size=1, [0]={_Val._Val} }}</DisplayString>
```
![image](https://user-images.githubusercontent.com/35536269/115435692-63a40b80-a20a-11eb-8d3b-140e1c4992a7.png)

Reply:
I think is not necessary to emphasize since the debugger already tell you the type, also I think its more useful to see the information. I still get your point, don't do any change yet, I would like to hear another maintainer opinion first.

Reply:
@CaseyCarter What do you think?

Reply:
I agree with @AnjuDel that we don't really need the reminder that `single_view` has a size of 1, and that folks are orders of magnitude more likely to want to see the contained value. I would keep the braces in the display, however, to keep it looking like "thing that contains `42`" instead of simply "`42`":
```suggestion
    <DisplayString>{{ _Val._Val }}</DisplayString>
```